src/glados/static/coffee/compoundReportCardApp.coffee

```diff
@@ -113,6 +113,7 @@ class CompoundReportCardApp extends glados.ReportCardApp
         else
           return true
       properties_to_show: Compound.COLUMNS_SETTINGS.COMPOUND_SOURCES_SECTION
+      sort_alpha: true
```

Comment:
it will always be the first property? this view also is used in other contexts, see for example the biocomponents section of:
https://chembl-glados.herokuapp.com/compound_report_card/CHEMBL1201585
it can be used for different properties shown in different ways